apps/address-book/arapp.json

```diff
@@ -42,8 +42,7 @@
     },
     "production": {
       "registry": "0x314159265dd8dbb310642f98f50c066173c1259b",
-      "appName": "address-book.open.aragonpm.eth",
-      "wsRPC": "wss://mainnet.eth.aragon.network/ws",
```

Comment:
What's the reasoning behind removing `wsRPC`?

Reply:
The connections and deploys fails with that config in there. it gets the endpoint from either the network or elsewhere